src/main/kotlin/pl/jutupe/ktor_rabbitmq/RabbitConsumer.kt

```diff
@@ -14,17 +16,21 @@ fun Application.rabbitConsumer(configuration: RabbitMQ.() -> Unit): RabbitMQ =
 inline fun <reified T> RabbitMQ.consume(
     queue: String,
     autoAck: Boolean = true,
-    crossinline rabbitDeliverCallback: (consumerTag: String, body: T) -> Unit
+    crossinline rabbitDeliverCallback: (consumerTag: String, body: T, channel: Channel, envelope: Envelope) -> Unit,
```

Comment:
Added parameters necessary for manual acking, not sure there is a proper way to do so otherwise

Reply:
Hmm, I will try to implement something, that does not expose channel and envelope to consumer, i think that we need to keep it as simple as possible